neo4j/README.md

```diff
@@ -3,11 +3,14 @@
 ## Overview
 
 This check monitors [neo4j][1] through the Datadog Agent.
+In [Neo4j 5][9], all metric names now include dbms or database namespaces.
+
+This check has been tested against Neo4j 4 and Neo4j 5 databases. If there are any issues, contact DataDog support or support@neo4j.com
```

Comment:
```suggestion
This check has been tested against Neo4j 4 and Neo4j 5 databases. If there are any issues, contact DataDog support or support@neo4j.com.
```

Reply:
@davidfauth wanted to bump this for you :)